### ISSUE TITLE
The specified value '!' does not conform to the required format.

### ISSUE THREAD

**Issue by NetbearSoftware** [bgrins/spectrum#291]
Hello,

I've been using Spectrum for around 6 months without issue, however two days ago I noticed a javascript warning being thrown on all pages of my site.

> The specified value '!' does not conform to the required format.  The format is '#rrggbb' where rr, gg, bb are two-digit hexadecimal numbers.

When I remove the spectrum plugin the warning goes away. This warning appears on pages where I am not initializing the plugin, just including it.

When I click on the warning it takes me to line 5595 of jQuery 1.11.2 which is the last line of the following code block:

```
// Deserialize a standard representation
tag = (rtagName.exec( elem ) || [ "", "" ])[ 1 ].toLowerCase();
wrap = wrapMap[ tag ] || wrapMap._default;

tmp.innerHTML = wrap[1] + elem.replace( rxhtmlTag, "<$1></$2>" ) + wrap[2];
```

I'm using Chrome 41.0.2272.76.

Appreciate any advice you can give and thank-you for creating such an amazing plugin!

Tim

**Comment by wolffe** [comment]
I have fixed it by replacing the "!" with "#ffffff" at line 75 of spectrum.js:

`
inputTypeColorSupport = (function() {
var colorInput = $("<input type='color' value='#ffffff' />")[0];
return colorInput.type === "color" && colorInput.value !== "#ffffff";
})(),
`

**Comment by NetbearSoftware** [comment]
Thank-you!

I'll close this issue seeing it looks like it has been resolved in v1.6.2.
